packaging/crosswalk-libs.spec

```diff
@@ -225,6 +225,10 @@ DEPENDENCIES=`python %{SOURCE1002} dump.json ${NINJA_TARGETS}`
 
 ninja %{?_smp_mflags} -C src/out/Release ${DEPENDENCIES}
 
+%if %{?BUILDDIR_NAME:1}%{!?BUILDDIR_NAME:0}
```

Comment:
Also, if you're doing this it makes sense to add a comment there explaining why you're removing a seemingly random file from the build directory.